demos/demo-react-ts/src/components/screens/KeypadScreen.tsx

```diff
@@ -83,47 +96,73 @@ function KeypadScreen({
   };
 
   const addToDialNumber = (value: string) => {
-    handleSetDialNumber(dialNumber + value);
+    handleSetDialNumber(toNumber + value);
     dialNumberInput.current?.focus();
   };
 
   const handleOutgoingCall = useCallback(() => {
     const callStartTime = Date.now();
     cti.outgoingCall({
       createEngagement: true,
-      phoneNumber: dialNumber,
+      phoneNumber: toNumber,
       callStartTime,
     });
     startTimer(callStartTime);
+    setDirection("OUTBOUND");
     handleNextScreen();
-  }, [cti, dialNumber, handleNextScreen, startTimer]);
+  }, [cti, toNumber, handleNextScreen, startTimer, setDirection]);
+
+  const handleTriggerIncomingCall = useCallback(
+    (incomingCallNumber: string) => {
+      cti.incomingCall({
+        createEngagement: true,
+        fromNumber: incomingCallNumber,
+        toNumber: fromNumber,
+      });
```

Comment:
Why are we calling the `fromNumber` instead of the `toNumber`? Also I think it is better to use `dialNumber` rather than `toNumber` to avoid confusion for inbound and outbound

Reply:
Usually toNumber/dialNumber is empty. If we pass in empty number to 'toNumber' property, caller id match call fails. I can revert toNumber baclk to dialNumber

Reply:
Done